README-Accumulation.md

```diff
@@ -0,0 +1,58 @@
+# Advanced-Reports: Accumulation Reports
+
+## Using Calculation Logic
+
+In the options which support calculation logic, you can use any REDCap calculation logic/functions
+and Smart Variables. Additionally the `[accumulation-number]` Smart Variable is supported in the
+*accumulation logic*, *group by* and *custom label logic* options.
+
+## Accumulation range
+
+The accumulation range determines the first and last accumulation number, as well as the step which
+increments or decrements the accumulation number by the specified amount. If the start number is
+less than the end number, the accumulation number is incremented by the step amount each time. If
+the start number is greater than the end number, the accumulation number is decremented by the step
+amount each time.
+
+The step amount is a number greater than 0. If not set it defaults to 1.
+
+The start and end values accept REDCap calculation logic. This may be useful for generating a range
+based on dates, e.g. using logic such as `year('today')`.
+
+## Accumulation logic
+
+For each record and accumulation number, the accumulation logic will be run to get a result number.
+These results will be summed together to obtain the cumulative value which is used in the report.
+
+## Group by
+
+If logic is entered here, it will be run for each record to get a *group name* which is used to
+group the records together. The cumulative values will be split by group in the report.
+
+## Display format
```

Comment:
Just so as you have a record of my request here, not a change for this version - I think it would be nice to have the 'Display format' option not as part of the report definition, but as a selectable option by the report user when looking at the report data.  Or perhaps both... in report definition it could be the 'default' format, that could be changed by the viewer when report is opened?

Reply:
This will be considered in future versions.